src/test/java/org/humancellatlas/ingest/SubmissionStateUpdaterTest.java

```diff
@@ -175,7 +174,7 @@ public void testUpdateSubmissionState() throws Exception {
 
         assertEquals(
                 ingestApiClient.retrieveSubmissionEnvelope(submissionEnvelopeReference).getSubmissionState().toUpperCase(),
-                SubmissionState.VALID.toString().toUpperCase());
+                SubmissionState.METADATA_VALID.toString().toUpperCase());
```

Comment:
could you also extend the test to also do the transitions for graph states?

Reply:
I have just renamed the state to be `GRAPH_VALID`. I was trying to extend it to fail from `METADATA_VALID` -> `GRAPH_VALID` and then do the proper transitions between graph states but I realised that the SubmissionStateUpdater is only responsible for persisting state changes in core, not dealing with incorrect transitions

Reply:
I think it would still be good to add those transitions in the test even just the happy path to test the state machine configuration changes.

Reply:
@aaclan-ebi tests for that are here https://github.com/ebi-ait/ingest-state-tracking/pull/9/files#diff-cec8d151dcefbc3975543d17843f377d41d306bbe4b5e1b30d38d09dee359188R119

Reply:
Aww, i missed them because they weren't loaded by default as it's a large diff.